src/content/docs/en/pages/guides/edge-firewall/edge-firewall.mdx

```diff
@@ -27,7 +27,7 @@ Before you begin, make sure that the **Edge Firewall** service and its modules a
 
 To show how it works, we will configure **Edge Firewall** for an e-commerce store (`www.myecommerce.com`) as an example, and demonstrate how we will prevent a particular malicious website from redirecting traffic from our store. For this, we will configure a rule within **Edge Firewall** to block *Header HTTP Referer*.
 
-From **Azion Console**, create a new configuration of **Edge Firewall** or edit an existing one. We will need to include your domain on the *Main Settings* tab, so that it can be protected by the rule. Save your rule set and next add a *New Rule* on the *Rules Engine* tab. Choose a name in the *Rule Engine* and define the activation criteria (Criteria). 
+From **Azion Console**, create a new configuration of **Edge Firewall** or edit an existing one. We will need to include your domain on the *Main Settings* tab, so that it can be protected by the rule. Save your edge firewall and next add a *New Rule* on the *Rules Engine* tab. Choose a name in the *Rule Engine* and define the activation criteria (Criteria). 
```

Comment:
```suggestion
From **Azion Console**, create a new configuration of **Edge Firewall** or edit an existing one. You'll need to include your domain on the *Main Settings* tab, so that it can be protected by the rule. Save your edge firewall and next add a *New Rule* on the *Rules Engine* tab. Choose a name in the *Rule Engine* and define the activation criteria (Criteria). 
```